bundles/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/live/TelldusLiveDeviceController.java

```diff
@@ -93,20 +95,24 @@ public TelldusLiveDeviceController() {
 
     @Override
     public void dispose() {
-        try {
-            client.close();
-        } catch (Exception e) {
-            logger.debug("Failed to close client", e);
+        AsyncHttpClient client = this.client;
+        if (client != null) {
+            try {
+                client.close();
+            } catch (Exception e) {
+                logger.debug("Failed to close client", e);
+            }
         }
     }
 
     void connectHttpClient(String publicKey, String privateKey, String token, String tokenSecret) {
         ConsumerKey consumer = new ConsumerKey(publicKey, privateKey);
         RequestToken user = new RequestToken(token, tokenSecret);
         OAuthSignatureCalculator calc = new OAuthSignatureCalculator(consumer, user);
-        this.client = new DefaultAsyncHttpClient(createAsyncHttpClientConfig());
+        AsyncHttpClient client = new DefaultAsyncHttpClient(createAsyncHttpClientConfig());
+        client.setSignatureCalculator(calc);
+        this.client = client;
```

Comment:
Optional, could be shortened:
```suggestion
        AsyncHttpClient client = this.client = new DefaultAsyncHttpClient(createAsyncHttpClientConfig());
        client.setSignatureCalculator(calc);
```